### PR TITLE
Update golden metrics for INFRA-CONTAINER

### PR DESCRIPTION
**Pull request by areina**
### Relevant information

- Update queries to use sample events instead of the new Metric event
type. There are still some services consuming these queries that can't
handle the new format.
- Remove the `as` clause in the select fields.
- Fix the queries to use the same kind of metrics between different data
sources.

@naxhh @IreneMLC do we support the double event types syntax, ex: `from: ContainerSample, K8sContainerSample` ?

That's supported by nrql and nrdb, I hope we also support it here.

Thank you

### Checklist

* [ ] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [ ] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

**Review by cianBuckley — APPROVED** [review]

**Comment by cianBuckley** [comment]
Unfortunately, there are performance/scaling concerns related to the double event table scoping in these queries. We need to make some internal changes before being able to merge this through. We will keep you posted!

**Comment by naxhh** [comment]
Closing this one since we will not merge it as-is.

I would suggest using the [roll-up](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/golden_metrics.md#roll-up-entities) functionality if we still want to provide this feature.

**Pull request closed without merging**
